runtime/particle-spec.js

```diff
@@ -94,18 +95,20 @@ class ParticleSpec {
     return this.slots.get(slotName);
   }
 
+  get primaryVerb() { if (this.verbs.length > 0) return this.verbs[0]; }
```

Comment:
Nit. Should be formatted over multiple lines.

Reply:
done